distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -1679,6 +1683,102 @@ public void testAccessGroups() throws Exception {
     }
   }
 
+  @Test
+  public void testSecurityManagerPolicy() {
```

Comment:
❓ Should these be separate test cases where we expect an exception to come out as opposed to having to catch them with a try/catch?

Reply:
I personally find this more readable than having all the tests broken out. If there is protest, I can change it.

Reply:
I think we should have one method per test. If the first one fail, the second one is never tested.